platform/marvell-arm64/sonic-platform-nokia/debian/rules

```diff
@@ -28,6 +30,12 @@ clean:
 
 build:
 	(for mod in $(MODULE_DIRS); do \
+		cd $(MOD_SRC_DIR)/$${mod}/; \
+		rm -rf mrvl-prestera || true; \
+		git clone ${MRVL_PRESTERA_SRC_URL}; \
+		cd mrvl-prestera && git checkout ${MRVL_PRESTERA_SRC_TAG} && cd ..; \
```

Comment:
can you make this as a submodule?

Reply:
There is a problem with making this as submodule. This git clone is done in "sonic-platform-nokia/7215/mrvl-prestera". When we introduce more "sonic-platform-<>/<mod>" we need to create same submodule in each of these. Currently we control versioning of this git tag present in "marvell-arm64/prestera.mk"(MRVL_PRESTERA_SRC_TAG). Do you still think this should be submodule?

Reply:
i do not understand this. this marvel asic platform driver, it should be like other asic driver on other platform, one set of driver for different devices. can you check like brcm/nvidia? not sure what you mean by sonic-platform-<>.

Reply:
@lguohan I have made the mrvl-prestera as submodule and pushed the changes. Please check.